packages/flutter/test/material/stepper_test.dart

```diff
@@ -1720,6 +1720,59 @@ testWidgets('Stepper custom indexed controls test', (WidgetTester tester) async
     ));
     expect(lastConnector.width, equals(0.0));
   });
+
+  // This is a regression test for https://github.com/flutter/flutter/issues/66007.
+  testWidgets('Vertical Stepper steps can be clipped', (WidgetTester tester) async {
```

Comment:
Should we write a test for horizontal as well?

Reply:
It's actually incorrect tittle. Initially I planned to test vertical and horizontal in two separate but it was too simple to split into two tests. Corrected the title. :)